src/libraries/System.Net.Http/src/System/Net/Http/Headers/NameValueHeaderValue.cs

```diff
@@ -17,10 +17,10 @@ public class NameValueHeaderValue : ICloneable
     {
         private static readonly Func<NameValueHeaderValue> s_defaultNameValueCreator = CreateNameValue;
 
-        private string? _name;
+        private string _name = null!; // Name always set after default constructor used
```

Comment:
Can we have:
```suggestion
        private string _name;
```

Reply:
No, this will make the compiler complain unless we explicitly assign a value in all constructors.